extensions/reviewed/FireBullet.json

```diff
@@ -1606,6 +1618,52 @@
           ],
           "objectGroups": []
         },
+        {
+          "description": "Change the speed variance (pixels per second) applied to each bullet.",
+          "fullName": "Set bullet speed variance",
+          "functionType": "Action",
```

Comment:
I suggest to use the action with operator for this kind of action as the change can't be done without keeping the old action:
- https://github.com/4ian/GDevelop/discussions/4533

It can be generated automagically so it's both faster to do and is more user friendly.